crates/bevy_transform/src/components/parent.rs

```diff
@@ -7,6 +7,7 @@ use bevy_ecs::{
 use bevy_reflect::Reflect;
 use std::ops::{Deref, DerefMut};
 
+/// Holds a reference to the parent entity of this entity, should only be present on entities that actually have a parent entity
```

Comment:
I would split this up into two sentences for better readability.

```suggestion
/// Holds a reference to the parent entity of this entity.
/// This component should only be present on entities that actually have a parent entity.
```